views/js/qtiCreator/widgets/interactions/helpers/bgImage.js

```diff
@@ -13,84 +13,32 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ * Copyright (c) 2015-2023 (original work) Open Assessment Technologies SA ;
  *
  */
 
 /**
  *
  * @author dieter <dieter@taotesting.com>
  */
-define([
-    'util/url',
-    'util/image',
-    'ui/mediasizer'
-], function (url, imageUtil) {
+define(['util/url', 'util/image', 'ui/mediasizer'], function (url, imageUtil) {
     'use strict';
-
-    /**
-     * @exports
-     */
-
-    /**
-     * Handle images that are larger than the canvas
-     *
-     * @param widget
-     */
-    function handleOversizedImages(widget) {
```

Comment:
This function wasn't used anywhere that's right? I checked in code base but better ask if this removal was intentional :)

Reply:
it is inner function, that no need now